src/view/services/command.manager.js

```diff
@@ -0,0 +1,12 @@
+export class CommandManager {
+	constructor(apply) {
+		this.apply = apply;
+	}
+
+	execute(commands) {
+		// First we need to get list of executable commands, cause execution of prev command can
```

Comment:
I think it's correct behavior, when a command affects possibility to execute of the next command. Maybe we need to execute them sequentially?